integrations/trusonafication_spec.rb

```diff
@@ -40,16 +42,29 @@
       end
     end
   end
-  describe 'creating a trusonafication for an unknown trusona user' do
+  describe 'creating a trusonafication without a level' do
     it 'as expected, does not work' do
-      @parameters[:email] = "#{Time.now.to_i}@example.com"
-
       expect {
         Trusona::Trusonafication.create(
           params: @parameters, timeout: @timeout
         )
       }.to raise_error(Trusona::InvalidResourceError)
     end
   end
+  describe 'creating a trusonafication with a callback url' do
+    it 'should POST to the URL when the trusonafication is completed' do
+      callback_id = SecureRandom.uuid
+
+      @parameters[:callback_url] = @buster.callback_url(callback_id)
+      @parameters[:expires_at] = DateTime.now + (1 / (24 * 60 * 60)) # one second
```

Comment:
I wonder if it's worth requiring `require 'active_support/core_ext/integer/time'` so that we can avoid the math for figuring out 1 second.

Reply:
this is where a joke would need to be made for the need to bring in a dependency to avoid simple math 🤷‍♂ 😄

Reply:
@thitu would prefer if we just wrote all of our code in binary... who needs higher level languages? I cry in my brain every time I'm not in Rails and need to do Date math. Since this is just a test I'm going to bring it in.... just to spite @thitu. I ❤️ you

Reply:
@thitu I would probably agree with you if it weren't for the comment required to help explain what the math is doing